src/sagemaker_containers/_modules.py

```diff
@@ -24,7 +24,9 @@
 import boto3
 import six
 from six.moves.urllib.parse import urlparse
+import yaml
```

Comment:
we do json for other config data. why introduce yaml?

Reply:
oh i see, we are using it for formatting output. I think we should just stick with native dict repr.

Reply:
Agreed, removed PYAML.